examples/resources/genesyscloud_flow_loglevel/resource.tf

```diff
@@ -0,0 +1,4 @@
+resource "genesyscloud_flow_loglevel" "flowLogLevel" {
```

Comment:
I would add every attribute the user can set to this example. It helps to show the full structure of the resource

Reply:
every possible flow log level or all the sub fields? The flowLogLevel characteristics can't be set individually, they're just set and returned based on the flow_log_level value a user provides